Library/Formula/chapel.rb

```diff
@@ -0,0 +1,47 @@
+class Chapel < Formula
+  homepage "http://chapel.cray.com/"
+  url "https://github.com/chapel-lang/chapel/releases/download/1.10.0/chapel-1.10.0.tar.gz"
+  sha1 "9c05c48f9309a7f685390df37753c086d1637c96"
+  head "https://github.com/chapel-lang/chapel.git"
+
+  def install
+    # Upstream claim chapel is parallel-build safe, but without the ENV compile = nope.
+    ENV.deparallelize
```

Comment:
The formula changed pretty significantly since this note was first inserted. There's a new note to remove the ` deparallelize ` with the next stable release, circa April 2015.